src/libcalamares/PythonHelper.cpp

```diff
@@ -94,9 +98,7 @@ variantListToPyList( const QVariantList& variantList )
 {
     bp::list pyList;
     foreach ( const QVariant& variant, variantList )
-    {
```

Comment:
I kinda liked these braces even for 1-line blocks. Keeps everyone from making mistakes if adding lines at some point.